irohad/main/impl/block_loader_init.cpp

```diff
@@ -30,7 +31,10 @@ auto BlockLoaderInit::createLoader(
     std::shared_ptr<BlockQuery> storage,
     std::shared_ptr<model::ModelCryptoProvider> crypto_provider) {
   return std::make_shared<BlockLoaderImpl>(
-      peer_query, storage, crypto_provider);
+      peer_query,
+      storage,
+      crypto_provider,
+      std::make_shared<shared_model::validation::BlockValidator>());
```

Comment:
this is your default value

Reply:
Yes, I want to do it explicitly because it can be changed in IR-837.